src/python/doc/persistence_graphical_tools_user.rst

```diff
@@ -90,3 +90,17 @@ If you want more information on a specific dimension, for instance:
     gudhi.plot_persistence_density(persistence=pers_diag,
         dimension=1, legend=True, axes=axes[1])
     plt.show()
+
+LaTeX support
+-------------
+
+By default, persistence graphical tools are using LaTeX support for matplotlib if available (cf.
+`matplotlib text rendering with LaTeX <https://matplotlib.org/3.3.0/tutorials/text/usetex.html>`_).
+It also requires `type1cm` LaTeX package (not detected by matplotlib).
+
+If you are facing issues with LaTeX rendering, you can still deactivate LaTeX rendering by saying:
```

Comment:
I think it would be nice to show here what the error message looks like, when type1cm is not available (skipping some parts since it is super long, but including at least the beginning and the end). Note that one of the issue reports was about type1ec.sty instead of type1cm.sty.
We could also mention that on Ubuntu, texlive-full (or more specific packages like texlive-latex-extra, cm-super) installs the dependencies.
Do we mention latex in the optional runtime dependencies on the installation page?

Reply:
Done on 9b52376e

> Do we mention latex in the optional runtime dependencies on the installation page?

The dependency is matplotlib, and LaTeX is a matplotlib dependency. I think it does not worth it to mention LaTeX in optional runtime dependencies.

Reply:
> The dependency is matplotlib, and LaTeX is a matplotlib dependency.

LaTeX is an optional dependency of matplotlib, which we explicitly use and thus kind of make a hard dependency. `apt install python3-matplotlib` does not install latex. That's why I was considering it. But ok.